ManageEngine/auditadplus/tests/test_iam_change_group.json

```diff
@@ -0,0 +1,41 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "AuditAD Plus",
+        "dialect_uuid": "890207d2-4878-440d-9079-3dd25d472e0a"
+      }
+    },
+    "message": "0|ManageEngine|ADAuditPlus|1|EventLog|GroupMgmtReports|1|cat=GroupMgmtReports cs1=Group Attributes Changed cs3=SERVER02.example.org type=member msg=Group 'MyGROUP' was modified by  'EXAMPLE\\J_DOE' Modified Properties : member, Values : CN\\=JANEDOE,OU\\=USERS,DC\\=example,DC\\=org rt=1694682151000 duser=MyGROUP sntdom=example.org duid=%{S-1-5-21-111111111-2222222222-3333333333-55555} suser=J_DOE cn1=1234 reason=Group Attribute Added cn2=1234567890 suid=S-1-5-21-111111111-2222222222-3333333333-44444"
+  },
+  "expected": {
+    "message": "0|ManageEngine|ADAuditPlus|1|EventLog|GroupMgmtReports|1|cat=GroupMgmtReports cs1=Group Attributes Changed cs3=SERVER02.example.org type=member msg=Group 'MyGROUP' was modified by  'EXAMPLE\\J_DOE' Modified Properties : member, Values : CN\\=JANEDOE,OU\\=USERS,DC\\=example,DC\\=org rt=1694682151000 duser=MyGROUP sntdom=example.org duid=%{S-1-5-21-111111111-2222222222-3333333333-55555} suser=J_DOE cn1=1234 reason=Group Attribute Added cn2=1234567890 suid=S-1-5-21-111111111-2222222222-3333333333-44444",
+    "event": {
+      "kind": "event",
+      "category": ["iam"],
+      "type": ["change"],
+      "module": "EventLog",
+      "severity": 1,
+      "dataset": "GroupMgmtReports",
+      "reason": "Group 'MyGROUP' was modified by  'EXAMPLE\\J_DOE' Modified Properties : member, Values : CN\\=JANEDOE,OU\\=USERS,DC\\=example,DC\\=org"
+    },
+    "@timestamp": "2023-09-14T09:02:31Z",
+    "observer": {
+      "vendor": "ManageEngine",
+      "product": "ADAuditPlus",
+      "version": "1",
+      "hostname": "SERVER02.example.org"
+    },
+    "user": {
+      "name": "J_DOE",
+      "target": {
+        "name": "MyGROUP",
+        "id": "%{S-1-5-21-111111111-2222222222-3333333333-55555}"
```

Comment:
Can you take a look to find a way to remove the bracket and the percent sign?
Thanks.